Azrux/Athenas-front
# Create footer

## Description
Create footer responsive

## Screenshots
![image](https://github.com/Azrux/Athenas-front/assets/89431511/c9ef8946-c982-4af3-b3ad-09c4f3800f10)
![image](https://github.com/Azrux/Athenas-front/assets/89431511/cc444ebe-c172-4752-b1e5-3535a2200af4)

## How to test?
* Run: `npm run dev`
* Go to [http://localhost:3000/test](http://localhost:3000/test)
* Check the footer is working
* Check the responsive design
